openshift/f8env.app.yaml

```diff
@@ -0,0 +1,151 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: f8env
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      service: f8env
+    name: f8env
+  spec:
+    replicas: ${{REPLICAS}}
+    selector:
+      service: f8env
+    strategy:
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          service: f8env
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          env:
+          - name: F8_POSTGRES_HOST
+            valueFrom:
+              secretKeyRef:
+                name: f8env
+                key: db.host
+          - name: F8_POSTGRES_PORT
+            valueFrom:
+              secretKeyRef:
+                name: f8env
+                key: db.port
+          - name: F8_POSTGRES_USER
+            valueFrom:
+              secretKeyRef:
+                name: f8env
+                key: db.user
+          - name: F8_POSTGRES_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: f8env
+                key: db.password
+          - name: F8_SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                name: f8env
+                key: sentry.dsn
+          - name: F8_POSTGRES_SSLMODE
+            valueFrom:
+              configMapKeyRef:
+                name: f8env
+                key: postgres.sslmode
+          - name: F8_POSTGRES_CONNECTION_MAXIDLE
+            valueFrom:
+              configMapKeyRef:
+                name: f8env
+                key: postgres.connection.maxidle
+          - name: F8_POSTGRES_CONNECTION_MAXOPEN
+            valueFrom:
+              configMapKeyRef:
+                name: f8env
+                key: postgres.connection.maxopen
+          - name: F8_ENVIRONMENT
+            valueFrom:
+              configMapKeyRef:
+                name: f8env
+                key: environment
+          - name: F8_AUTH_URL
+            valueFrom:
+              configMapKeyRef:
+                name: f8env
+                key: auth.url
+          imagePullPolicy: Always
+          name: f8env
+          ports:
+          - containerPort: 8080
```

Comment:
@alexeykazakov containerPort=8080 .. fine?

Reply:
Well.. using 8080 everywhere is fine for the service itself but for auth and cluster services we use different ports (8089 and 8087) just to make developer's live easier if they want to run a few services locally (not in minishift) in the same time.